network-api/app/networkapi/templates/pages/landingpage.html

```diff
@@ -7,7 +7,7 @@
     <meta property="og:type" content="website">
     <meta property="og:site_name" content="Mozilla Network">
     <meta property="og:locale" content="en_US">
-    <meta property="og:url" content="https://foundation.mozilla.orgnetwork-api/app/networkapi/templates/pages/landingpage.html">
+    <meta property="og:url" content="https://foundation.mozilla.org{{ page.get_absolute_url }}">
```

Comment:
This change gets nerfed whenever I run the project, should this file even be in git?